src/tr2/game/level.c

```diff
@@ -23,6 +23,7 @@
 #include <libtrx/game/level.h>
 #include <libtrx/log.h>
 #include <libtrx/memory.h>
+#include <libtrx/utils.h>
```

Comment:
I'm not sure we need this include – we don't seem to need anything from it in the new code.

Reply:
Thank you. I had a temporary need for `ABS` in https://github.com/LostArtefacts/TRX/pull/2340/commits/999b9a3f1ec4dce6118e80fd57b10ed96d88a7c9, and then promptly forgot to remove this in the shift to TRX.